src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -888,12 +888,11 @@ impl Step for Rustc {
 macro_rules! tool_doc {
     (
         $tool: ident,
-        $should_run: literal,
         $path: literal,
         $(rustc_tool = $rustc_tool:literal, )?
-        $(in_tree = $in_tree:literal ,)?
         $(is_library = $is_library:expr,)?
         $(crates = $crates:expr)?
+        $(, submodule $(= $submodule:literal)? )?
```

Comment:
Why isn't this just an optional argument, rather than a repeating argument?

Reply:
It's already an optional argument, not a repeating one

Reply:
Sorry :man_facepalming: I misread the macro.